Source/DafnyCore/Backends/Rust/Dafny-compiler-rust.dfy

```diff
@@ -4523,7 +2201,7 @@ module {:extern "DCOMP"} DafnyToRustCompiler {
               case Eq(referential) => {
                 if (referential) {
                   if pointerType.Raw? {
-                    r := Error("Cannot compare raw pointers yet - need to wrap them with a structure to ensure they are compared properly");
+                    r := Error("Raw pointer comparison not properly implemented yet (need to make comp/rust/traits.dfy to pass with --raw-pointer)");
```

Comment:
nit: It seems strange to mention this implementation detail in the error message.

```suggestion
                    // Need to make comp/rust/traits.dfy to pass with --raw-pointer)
                    r := Error("Raw pointer comparison not properly implemented yet");
```